panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.20 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.20 set. Notably, if go1.20 adds a moving garbage collector, this program is unsafe to use.  goroutine 1 [running]: go4.org/unsafe/assume-no-moving-gc.init.0()         /home/keeyu/go/pkg/mod/go4.org/unsafe/assume-no-moving-gc@v0.0.0-20220617031537-928513b29760/untested.go:25 +0x178

### 版本、安装方式、系统

1. 你在使用什么版本的v2rayA
v2raya-2.0.1-1
2. 你通过什么方式安装v2rayA
yay
3. 你所使用的操作系统
manjaro/arm64
### 描述问题
<!-- 在下方简要描述问题 -->
无法启动
运行直接报错
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.20 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.20 set. Notably, if go1.20 adds a moving garbage collector, this program is unsafe to use.  goroutine 1 [running]: go4.org/unsafe/assume-no-moving-gc.init.0()         /home/keeyu/go/pkg/mod/go4.org/unsafe/assume-no-moving-gc@v0.0.0-20220617031537-928513b29760/untested.go:25 +0x178
### 复现问题的步骤
<!-- 在下方描述如何复现问题 -->


### v2rayA/v2ray运行日志：

 ```bash
 # daemon mode
 journalctl -u v2raya | tail -n 50
 sudo tail -n 50 /var/log/v2raya/v2raya.log
 
 # docker mode
 docker logs v2raya
 ```
 或者粘贴来自 Web 页面的日志

```shell
# 此处贴入运行日志



```

### v2rayA设置以及相关截图
贴入v2rayA设置，以及问题相关的截图。
<!-- 如果条件允许请附图 -->

--- a/pkgs/tools/networking/v2raya/default.nix
+++ b/pkgs/tools/networking/v2raya/default.nix
@@ -10,13 +10,13 @@
 }:
 let
   pname = "v2raya";
-  version = "2.0.0";
+  version = "2.0.2";
 
   src = fetchFromGitHub {
     owner = "v2rayA";
     repo = "v2rayA";
     rev = "v${version}";
-    sha256 = "sha256-1fWcrMd+TSrlS1H0z7XwVCQzZAa8DAFtlekEZNRMAPA=";
+    sha256 = "sha256-C7N23s/GA66gQ5SVXtXcM9lXIjScR3pLYCrf0w2nHfY=";
   };
 
   web = mkYarnPackage {
@@ -45,7 +45,7 @@ buildGoModule {
   inherit pname version;
 
   src = "${src}/service";
-  vendorSha256 = "sha256-Ud4pwS0lz7zSTowg3gXNllfDyj8fu33H1L20szxPcOA=";
+  vendorSha256 = "sha256-vnhqI9G/p+SLLA4sre2wfmg1RKIYZmzeL0pSTbHb+Ck=";
 
   ldflags = [
     "-s"

